src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,243 @@
+                                            value: '',
+                                            label: 'Select your pronouns',
+                                        }}
+                                        value={this.state.pronouns}
+                                        Icon={() => <Icon src={DownArrow} />}
+                                    />
+                                </View>
+                                {this.state.pronouns === CONST.PRONOUNS.SELF_SELECT && (
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.selfSelectedPronouns}
+                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
+                                        placeholder="Self-select your pronoun"
+                                    />
+                                )}
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>
+                                    {Str.isSMSLogin(this.props.myPersonalDetails.login)
+                                        ? 'Phone Number' : 'Email Address'}
+                                </Text>
+                                <TextInput
+                                    style={[styles.textInput, styles.disabledTextInput]}
+                                    value={this.props.myPersonalDetails.login}
+                                    editable={false}
+                                />
+                            </View>
+                            <View style={styles.mb2}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Timezone</Text>
+                                <RNPickerSelect
+                                    onValueChange={selectedTimezone => this.setState({selectedTimezone})}
+                                    items={timezones}
+                                    style={this.state.isAutomaticTimezone ? {
+                                        ...styles.picker,
+                                        inputIOS: [styles.picker.inputIOS, styles.textInput, styles.disabledTextInput],
+                                        inputAndroid: [
+                                            styles.picker.inputAndroid, styles.textInput, styles.disabledTextInput,
+                                        ],
+                                        inputWeb: [styles.picker.inputWeb, styles.textInput, styles.disabledTextInput],
+                                    } : styles.picker}
+                                    useNativeAndroidPickerStyle={false}
+                                    value={this.state.selectedTimezone}
+                                    Icon={() => <Icon src={DownArrow} />}
+                                    disabled={this.state.isAutomaticTimezone}
```

Comment:
NAB: but the colors for disabled fields is different for timezones and primary logins
<img width="300" alt="" src="https://user-images.githubusercontent.com/16747822/111397742-3ef2cc00-867f-11eb-8d66-f9d26304394d.png">

cc: @shawnborton

Reply:
Good catch, ideally these look the same. I think the phone number example above is a bit more legible but still communicates that it is disabled.